src/EventList.js

```diff
@@ -65,7 +65,7 @@ export function EventList(props) {
     //Location filter
     if(props.locFilt != null){
       if('location' in event && 'location' in event['location'] && 'latitude' in event['location']['location']){
-        if(event['location']['location']['latitude'] !== props.locFilt['lat'] || event['location']['location']['longitude'] != props.locFilt['lng']){
+        if(event['location']['location']['latitude'] !== props.locFilt['lat'] || event['location']['location']['longitude'] !== props.locFilt['lng']){
```

Comment:
This event[location][location] is a bit wordy. Maybe consider processing the event upon arrival and either making it an instance of some ‘event’ class (would probably me cleanest), or otherwise simplify the object for easier access of the properties you tend to access.